bin/docker_build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Get the directory to build argument
+if [ -n "$1" ] && [ "$1" != "--rebuild" ]; then
+  directory_to_build=$1
+elif [ -n "$2" ] && [ "$2" != "--rebuild" ]; then
+  directory_to_build=$2
+fi
+
+# Build the Docker image, if the --rebuild flag was used or if an image hasn't been built yet
+
+if [ -z "$(docker images | grep wca-pdf-builder)" ]; then
+  image_exists=true
+  docker rmi wca-pdf-builder
+fi
+
+if [ "$1" = "--rebuild" ] || [ "$2" = "--rebuild" ] || [ "$image_exists" = true ]; then
```

Comment:
You can set a `$is_rebuilld` flag in the "Get the directory to build argument" block above, and then just do one "equal to true" check here

Reply:
But that one doesn't check for the flag being == to --rebuild, it checks the opposite along with another check. I'd have to add more if statements to do what you're suggesting. Would you like me to do that?

Reply:
Okay, I didn't consider the mumbo-jumbo'd way of appending the `--rebuild` argument in odd places. Nevermind,

Reply:
I figured it would be easier for the user if the order of the arguments didn't matter